packages/@aws-cdk-testing/framework-integ/test/aws-stepfunctions-tasks/test/bedrock/integ.invoke-model.ts

```diff
@@ -52,13 +52,32 @@ const prompt2 = new BedrockInvokeModel(stack, 'Prompt2', {
   resultPath: '$',
 });
 
+/** Test for Bedrock Output Path */
 const prompt3 = new BedrockInvokeModel(stack, 'Prompt3', {
   model,
-  inputPath: sfn.JsonPath.stringAt('$.names'),
+  body: sfn.TaskInput.fromObject(
+    {
+      inputText: sfn.JsonPath.format(
+        'Alphabetize this list of first names:\n{}',
+        sfn.JsonPath.stringAt('$.names'),
+      ),
+      textGenerationConfig: {
+        maxTokenCount: 100,
+        temperature: 1,
+      },
+    },
+  ),
   outputPath: sfn.JsonPath.stringAt('$.names'),
```

Comment:
Can help me to understand why body is required instead of inputPath?

Reply:
just added this integ test is to make sure, we are not breaking any existing use case for `outputPath` or `inputPath` from the base class, looking back at why it might have been marked as required could be a decision take while designing this s3 input parameter, i don't see it something in [API doc](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_InvokeModel.html)

Reply:
In order to test the inputPath field, you can do a Pass state to transform the previous output into the format expected for the InvokeModel body.